astropy/modeling/core.py

```diff
@@ -2747,6 +2791,47 @@ def __str__(self):
         ]
         return super()._format_str(keywords=keywords)
 
+    def _generate_input_output_units_dict(self, mapping, attr):
+        d = {}
+        for inp, (model, orig_inp) in mapping.items():
+            mattr = getattr(model, attr)
+            if isinstance(mattr, dict):
+                if orig_inp in mattr:
+                    d[inp] = mattr[orig_inp]
+            elif isinstance(mattr, bool):
+                d[inp] = mattr
+
+        if d:
+            return d
+
+    @property
+    def _supports_unit_fitting(self):
+        return False
```

Comment:
Just for the record, this already didn't work, we are just marking it explicitly as False until we can fix it in a separate PR